# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/ds/StaticAtoms.py
# Commit: 0a3da2090e48
# Full Hash: 0a3da2090e4859473f52b585bcb39baae2876b28
# Author: Jared Wein <jwein@mozilla.com>
# Date: 2021-02-26 03:55:47
# Regressor Bug: 1665390
# File Overlap Count: 1
# Description:
#   Bug 1665390 - Style bookmarks toolbar and tabs toolbar tooltips. r=NeilDeakin,ntim
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104257
# ==============================================================================

diff -r 176b788753e8 -r 0a3da2090e48 xpcom/ds/StaticAtoms.py
--- a/xpcom/ds/StaticAtoms.py	Thu Feb 25 20:47:15 2021 +0000
+++ b/xpcom/ds/StaticAtoms.py	Thu Feb 25 20:47:54 2021 +0000
@@ -100,6 +100,7 @@
     Atom("ancestor", "ancestor"),
     Atom("ancestorOrSelf", "ancestor-or-self"),
     Atom("anchor", "anchor"),
+    Atom("anchortoclosest", "anchortoclosest"),
     Atom("_and", "and"),
     Atom("animations", "animations"),
     Atom("anonid", "anonid"),
